.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks.git
+    rev: v1.4.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-ast
+      - id: check-case-conflict
+      - id: check-merge-conflict
+      - id: check-docstring-first
+      - id: trailing-whitespace
+      - id: check-symlinks
+  - repo: https://github.com/pre-commit/mirrors-yapf
+    rev: v0.27.0
+    hooks:
+      - id: yapf
```

Comment:
Use [black](https://github.com/psf/black) instead of yapf. I have had project codes goes to hell before with yapf.

For formatting md/yaml and other files I suggest prettier, but it's pretty much optional and would require contributors to have node installed.

Reply:
I already changed it to use black, any suggestion for the line length? (currently using black's default)

Reply:
Default is fine, although my personal preference is 120.

Reply:
I already changed it to 120.